app/build.gradle

```diff
@@ -4,7 +4,8 @@ plugins {
     id 'com.google.gms.google-services'
     id 'com.google.firebase.crashlytics'
 }
-
+def localProperties = new Properties()
+localProperties.load(new FileInputStream(rootProject.file("local.properties")))
```

Comment:
Please apply here same solution as Albert did:
https://github.com/SafetyHeads/Akademia-Androida-2023/pull/77/files#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9

This is fix for failing build at CI